frontend/src/metabase/public/containers/PublicOrEmbeddedDashboard/PublicOrEmbeddedDashboardView.stories.tsx

```diff
@@ -86,6 +91,9 @@ const CARD_TABLE_ID = getNextId();
 const TAB_ID = getNextId();
 const PARAMETER_ID = "param-hex";
 const initialState = createMockState({
+  settings: createMockSettingsState({
+    "hide-embed-branding?": false,
```

Comment:
We'll have [a new design](https://metaboat.slack.com/archives/C063Q3F1HPF/p1720078600140479) for your Metabase badge to support the transparent background soon. So I show Metabase logo here.